#!/usr/bin/env python
PACKAGE = "pepper_vr_teleop"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("head_yaw", double_t, 0, "Angle for the HeadYaw joint", 0, -2.0857, 2.0857)
gen.add("head_pitch", double_t, 0, "Angle for the HeadPitch joint", 0, -0.7068, 0.4451)
gen.add("l_shoulder_pitch", double_t, 0, "Angle for the LShoulderPitch joint", 0, -2.0857, 2.0857)
gen.add("l_shoulder_roll", double_t, 0, "Angle for the LShoulderRoll joint", 0, 0.0087, 1.562)
gen.add("l_elbow_yaw", double_t, 0, "Angle for the LElbowYaw joint", 0, -2.0857, 2.0857)
gen.add("l_elbow_roll", double_t, 0, "Angle for the LElbowRoll joint", 0, -1.562, -0.0087)
gen.add("r_shoulder_pitch", double_t, 0, "Angle for the RShoulderPitch joint", 0, -2.0857, 2.0857)
gen.add("r_shoulder_roll", double_t, 0, "Angle for the RShoulderRoll joint", 0, -1.562, -0.0087)
gen.add("r_elbow_yaw", double_t, 0, "Angle for the RElbowYaw joint", 0, -2.0857, 2.0857)
gen.add("r_elbow_roll", double_t, 0, "Angle for the RElbowRoll joint", 0, 0.0087, 1.562)

exit(gen.generate(PACKAGE, "pepper_vr_teleop", "TestControl"))
